.rubocop_todo.yml

```diff
@@ -1,11 +1,28 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config --auto-gen-only-exclude --exclude-limit 99999`
-# on 2021-03-22 18:46:17 UTC using RuboCop version 1.11.0.
+# on 2022-02-16 17:51:47 UTC using RuboCop version 1.25.1.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
+# Offense count: 13
+InternalAffairs/InheritDeprecatedCopClass:
```

Comment:
Newer rubocop wants us to inherit from a new `Base` class instead of `Cop`, but there are breaking API changes. I started working through them, but decided to pull that out to a separate branch. We don't need to resolve these in order to get a gem release out.